minimesos/src/main/java/com/containersol/minimesos/util/Environment.java

```diff
@@ -0,0 +1,25 @@
+package com.containersol.minimesos.util;
+
+/**
+ * Utility for detecting the runtime environment.
+ */
+public class Environment {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AS1118)